extension/chrome/elements/compose-modules/compose-render-module.ts

```diff
@@ -32,7 +31,7 @@ export class ComposeRenderModule extends ViewModule<ComposeView> {
     }
     await this.view.replyPopoverModule.render(this.view.isReplyBox);
     this.initComposeBoxStyles();
-    if (!this.view.draftId && await this.view.draftModule.localDraftGet()) {
+    if (!this.view.draftId && (await this.view.draftModule.localDraftGet())) {
```

Comment:
I'd say this one is even more questionable, `&& await something` looks legit to me and better than `&& (await something)` but since I don't contribute that much here anymore, I'll follow what you think

Reply:
Yeah, here `await something` looks cleaner, but I think formatter automatically adds brackets to be consistent for all `await` expressions and to avoid unexpected behavior when, for example, in the future we'll change code to something like
```js
await this.view.draftModule.localDraftGet().id

// which will work only with brackets 
(await this.view.draftModule.localDraftGet()).id
```

Reply:
This behavior is not configurable whatsoever?

Reply:
Yes, I didn't find exact option for removing these brackets

Reply:
Alright